### DISCUSSION TITLE
멀미가 납니다.

### DISCUSSION THREAD

**Discussion by jihwanKo** [metamong-friends/metamong-friends.github.io#15] [제안하기]
쿼터뷰에서 이동할때 멀미가 납니다.
원인을 정확히 설명하기가 어려운데요.. 
이동시작할때 카메라가 살짝씩 움직이는 거 때문에 그런게 아닐까 싶습니다.

**Comment by jihwanKo** [comment]
참고로 저는 원래 멀미에 민감합니다. FPS 게임같은거 멀미나서 잘 몬합니다.

**Reply by shlee303** [reply]
이 부분은 저희도 3D 이동에 대해 고민하고 있는데, 좀 더 자연스러운 카메라 동작을 연구해보겠습니다.

**Comment by shlee303** [comment]
@jihwanKo 카메라 무브먼트 개선 및, 우클릭으로 방향 전환 시 고정되도록 조치 완료 하였습니다. 확인해보시고 혹시 아직 흔들리는 이슈가 있다면 코멘트 부탁드려요~! 감사합니다!
